Bottom: a52fff43a00ef7782f2bed16ecc75238cfab664d
Top:    fa8bf236032a6911f02ffecf40da83d4f5171e73
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-02 13:02:22 +0000

hid/common: Fix including draw_helpers.h from draw_helpers.c

The prototype for hid_common_fill_pcb_pv() was incorrect. To avoid this
kind of mistake in the future, include the header file from the .c file.


---

diff --git a/src/hid/common/draw_helpers.c b/src/hid/common/draw_helpers.c
index f6fbe71..02473be 100644
--- a/src/hid/common/draw_helpers.c
+++ b/src/hid/common/draw_helpers.c
@@ -4,6 +4,7 @@
 #include "data.h" /* For global "PCB" variable */
 #include "rotate.h" /* For RotateLineLowLevel() */
 #include "polygon.h"
+#include "draw_helpers.h"
 
 
 static void
diff --git a/src/hid/common/draw_helpers.h b/src/hid/common/draw_helpers.h
index 1e6cb7d..95c2dc1 100644
--- a/src/hid/common/draw_helpers.h
+++ b/src/hid/common/draw_helpers.h
@@ -4,6 +4,6 @@ void common_thindraw_pcb_polygon (hidGC gc, PolygonType *poly,
                                   const BoxType *clip_box);
 void common_fill_pcb_pad (hidGC gc, PadType *pad, bool clear, bool mask);
 void common_thindraw_pcb_pad (hidGC gc, PadType *pad, bool clear, bool mask);
-void common_fill_pcb_pv (hidGC gc, PinType *pv, bool drawHole, bool mask);
+void common_fill_pcb_pv (hidGC fg_gc, hidGC bg_gc, PinType *pv, bool drawHole, bool mask);
 void common_thindraw_pcb_pv (hidGC fg_gc, hidGC bg_gc, PinType *pv, bool drawHole, bool mask);
 void common_draw_helpers_init (HID_DRAW *graphics);
